extension/chrome/src/js/.eslintrc

```diff
@@ -0,0 +1,23 @@
+{
+    "extends": "eslint:recommended",
+    "env": {
+        "browser": true,
+        "es6": true,
+        "webextensions": true
+    },
+    "globals": {
+        "extensionStorage": false,
+        "jsonGroups": true,
+        "jQuery": false,
+        "$": false,
```

Comment:
The stash_page.js still uses it since it pulls it from bitbucket using require.js; the goal was to remove the jQuery instance bundled with the extension.